pages/ksphere/kommander/latest/operations/access-control/index.md

```diff
@@ -6,36 +6,30 @@ menuWeight: 1
 excerpt: Centrally manage access across clusters
 ---
 
-Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters. The resources are similar to Kubernetes RBAC but with some crucial differences.
+Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters or a subset of clusters. The resources may seem familiar because they are similar to Kubernetes RBAC but with some crucial differences that make it possible to define the roles and policies once and have them federated to clusters within a given scope:
```

Comment:
Role-based authorization can be defined centrally within Kommander to control resource access on all clusters or a subset of clusters. The resources are similar to Kubernetes RBAC but with crucial differences. These resouces  make it possible to define the roles and policies once and have them federated to clusters within a given scope: